playwright/seccomp_profile.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
this shouldnt be here :(

Reply:
Weird. Might be because the /dev branch was outdated when I forked. I'll look at it later.